CHANGELOG.md

```diff
@@ -30,7 +30,8 @@
 ### 🛠 Maintenance
 
 - Update caniuse database ([#1046](https://github.com/opensearch-project/oui/pull/1046))
-- Bump TypeScript to v4.6.4 ([#879](https://github.com/opensearch-project/oui/pull/879))
+- Bump TypeScript to v4.6.4 ([#879](https://github.com/opensearch-project/oui/pull/879)
```

Comment:
```suggestion
- Bump TypeScript to v4.6.4 ([#879](https://github.com/opensearch-project/oui/pull/879))
```

Looks like a `)` got cut off here accidentally

Reply:
@BSFishy fixed the error.